Build fails but in the search service because of the spring failure.
```
[ERROR] Failed to execute goal on project opencast-search-service-impl: Could not resolve dependencies for project org.opencastproject:opencast-search-service-impl:bundle:9-SNAPSHOT: Failed to collect dependencies at org.jdom:com.springsource.org.jdom:jar:2.0.1: Failed to read artifact descriptor for org.jdom:com.springsource.org.jdom:jar:2.0.1: Could not transfer artifact org.jdom:com.springsource.org.jdom:pom:2.0.1 from/to spring.io (https://repo.spring.io/plugins-release/): Not authorized , ReasonPhrase:Unauthorized. -> [Help 1]
[ERROR] 
```

URL that fails: `https://repo.spring.io/plugins-release/org/jdom/com.springsource.org.jdom/2.0.1/com.springsource.org.jdom-2.0.1.pom`

--- a/assemblies/karaf-features/src/main/feature/feature.xml
+++ b/assemblies/karaf-features/src/main/feature/feature.xml
@@ -72,10 +72,6 @@
     <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.xmlschema/1.4.3_1</bundle>
     <bundle>mvn:org.codehaus.jettison/jettison/1.4.0</bundle>
     <bundle>mvn:org.codehaus.groovy/groovy/2.4.3</bundle>
-    <!-- At least ingest-service-impl need JDOM in version 1.x -->
-    <!-- JDOM 1.x updates will be bug fixes as new features are gone to 2.x (see http://www.jdom.org/) -->
-    <bundle>mvn:org.jdom/com.springsource.org.jdom/1.1.0</bundle>
-    <!-- At least search-service-impl need JDOM in version 2.x -->
     <bundle>mvn:org.jdom/com.springsource.org.jdom/2.0.1</bundle>
     <bundle>mvn:org.opencastproject/android-mms/1.2</bundle>
     <bundle>wrap:mvn:com.googlecode.json-simple/json-simple/${json-simple.version}</bundle>

--- a/docs/checkstyle/maven-dependency-plugin.exceptions
+++ b/docs/checkstyle/maven-dependency-plugin.exceptions
@@ -8,7 +8,6 @@ modules/db/pom.xml
 modules/distribution-service-aws-s3/pom.xml
 modules/external-api/pom.xml
 modules/index-service/pom.xml
-modules/ingest-service-impl/pom.xml
 modules/oaipmh-persistence/pom.xml
 modules/publication-service-youtube-v3/pom.xml
 modules/scheduler-impl/pom.xml

--- a/modules/ingest-service-impl/pom.xml
+++ b/modules/ingest-service-impl/pom.xml
@@ -69,10 +69,6 @@
       <artifactId>javax.servlet-api</artifactId>
     </dependency>
     <dependency>
-      <groupId>jakarta.xml.bind</groupId>
-      <artifactId>jakarta.xml.bind-api</artifactId>
-    </dependency>
-    <dependency>
       <groupId>jakarta.ws.rs</groupId>
       <artifactId>jakarta.ws.rs-api</artifactId>
     </dependency>
@@ -128,6 +124,19 @@
       <artifactId>commons-fileupload</artifactId>
     </dependency>
     <dependency>
+      <groupId>com.entwinemedia.common</groupId>
+      <artifactId>functional</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore-osgi</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient-osgi</artifactId>
+    </dependency>
+
+    <dependency>
       <groupId>org.opencastproject</groupId>
       <artifactId>opencast-working-file-repository-service-impl</artifactId>
       <version>${project.version}</version>
@@ -147,13 +156,20 @@
         </exclusion>
       </exclusions>
     </dependency>
-    <dependency>
-      <groupId>org.jdom</groupId>
-      <artifactId>com.springsource.org.jdom</artifactId>
-    </dependency>
   </dependencies>
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <ignoredUnusedDeclaredDependencies>
+            <!-- provide a logger for tests -->
+            <ignoredUnusedDeclaredDependency>org.slf4j:slf4j-log4j12</ignoredUnusedDeclaredDependency>
+          </ignoredUnusedDeclaredDependencies>
+        </configuration>
+      </plugin>
       <plugin>
         <groupId>org.apache.felix</groupId>
         <artifactId>maven-bundle-plugin</artifactId>

--- a/modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java
+++ b/modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java
@@ -97,13 +97,6 @@ import org.apache.cxf.jaxrs.ext.multipart.ContentDisposition;
 import org.apache.http.Header;
 import org.apache.http.HttpResponse;
 import org.apache.http.client.methods.HttpGet;
-import org.jdom.Document;
-import org.jdom.Element;
-import org.jdom.JDOMException;
-import org.jdom.Namespace;
-import org.jdom.filter.ElementFilter;
-import org.jdom.input.SAXBuilder;
-import org.jdom.output.XMLOutputter;
 import org.osgi.service.cm.ManagedService;
 import org.osgi.service.component.ComponentContext;
 import org.slf4j.Logger;
@@ -111,17 +104,15 @@ import org.slf4j.LoggerFactory;
 
 import java.beans.PropertyChangeEvent;
 import java.beans.PropertyChangeListener;
-import java.io.ByteArrayInputStream;
-import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URI;
+import java.nio.charset.StandardCharsets;
 import java.util.Arrays;
 import java.util.Date;
 import java.util.Dictionary;
 import java.util.HashMap;
 import java.util.HashSet;
-import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
@@ -471,8 +462,9 @@ public class IngestServiceImpl extends AbstractJobProducer implements IngestServ
             continue;
 
           if (entry.getName().endsWith("manifest.xml") || entry.getName().endsWith("index.xml")) {
-            // Build the mediapackage
-            mp = loadMediaPackageFromManifest(new ZipEntryInputStream(zis, entry.getSize()));
+            // Build the media package
+            final InputStream is = new ZipEntryInputStream(zis, entry.getSize());
+            mp = MediaPackageParser.getFromXml(IOUtils.toString(is, StandardCharsets.UTF_8));
           } else {
             logger.info("Storing zip entry {}/{} in working file repository collection '{}'", job.getId(),
                     entry.getName(), wfrCollectionId);
@@ -564,50 +556,6 @@ public class IngestServiceImpl extends AbstractJobProducer implements IngestServ
         workingFileRepository.deleteFromCollection(Long.toString(job.getId()), filename, true);
       }
     }
-  }
-
-  private MediaPackage loadMediaPackageFromManifest(InputStream manifest)
-          throws IOException, MediaPackageException, IngestException {
-    // TODO: Uncomment the following line and remove the patch when the compatibility with pre-1.4 MediaPackages is
-    // discarded
-    //
-    // mp = builder.loadFromXml(manifestStream);
-    //
-    // =========================================================================================
-    // =================================== PATCH BEGIN =========================================
-    // =========================================================================================
-    ByteArrayOutputStream baos = null;
-    ByteArrayInputStream bais = null;
-    try {
-      Document domMP = new SAXBuilder().build(manifest);
-      String mpNSUri = "http://mediapackage.opencastproject.org";
-
-      Namespace oldNS = domMP.getRootElement().getNamespace();
-      Namespace newNS = Namespace.getNamespace(oldNS.getPrefix(), mpNSUri);
-
-      if (!newNS.equals(oldNS)) {
-        @SuppressWarnings("rawtypes")
-        Iterator it = domMP.getDescendants(new ElementFilter(oldNS));
-        while (it.hasNext()) {
-          Element elem = (Element) it.next();
-          elem.setNamespace(newNS);
-        }
-      }
-
-      baos = new ByteArrayOutputStream();
-      new XMLOutputter().output(domMP, baos);
-      bais = new ByteArrayInputStream(baos.toByteArray());
-      return MediaPackageParser.getFromXml(IOUtils.toString(bais, "UTF-8"));
-    } catch (JDOMException e) {
-      throw new IngestException("Error unmarshalling mediapackage", e);
-    } finally {
-      IOUtils.closeQuietly(bais);
-      IOUtils.closeQuietly(baos);
-      IOUtils.closeQuietly(manifest);
-    }
-    // =========================================================================================
-    // =================================== PATCH END ===========================================
-    // =========================================================================================
   }
 
   /**

